### DISCUSSION TITLE
[Notice]Thanks for your valuable contribution！

### DISCUSSION THREAD

**Discussion by debrachena** [apache/dolphinscheduler#7764] [Show and tell]
Hi Developers,

Wish you all a brand-new start in 2022!

In the past year, the COVID-19 pandemic didn't stop the contribution enthusiasm of the community members

Apache DolphinScheduler had made so many breakthroughs

These cannot be achieved without the contributions of the community

On the moment to welcome the new year

A delicate gift will be sent to all community contributors!

Anyone who have participated in the contribution can receive the gift after indicating your GitHub ID and submitting a record to verify your identity and contribution. Non-code contributors can also mark your GitHub ID and contribution record in the comment area.

Example: GitHub ID: xxx

Contribution link:

Deadline: January 19, 2022

Tips:

Don't forget to fill out the questionnaire to get the gift.
https://jinshuju.net/f/zDyzxp

**Comment by Lucaszlei** [comment]
GitHub ID: SunnyZ-L 
Contribution link: 
https://github.com/apache/dolphinscheduler/pull/7634
https://github.com/apache/dolphinscheduler/pull/6947

**Comment by ruanwenjun** [comment]
GitHub ID: ruanwenjun
Contribution link: #7556

**Comment by uh001** [comment]
GitHub ID: uh001
Contribution link:
#7761
#7745

**Comment by Squidyu** [comment]
GitHub ID: Squidyu
Contribution link: https://github.com/apache/dolphinscheduler-website/pull/425
[[#5936](https://github.com/apache/dolphinscheduler/pull/5936)](https://github.com/apache/dolphinscheduler/pull/5936)
https://github.com/apache/dolphinscheduler/pull/5682
https://github.com/apache/dolphinscheduler/pull/5574
https://github.com/apache/dolphinscheduler/pull/5573

**Comment by QuakeWang** [comment]
GitHub ID: QuakeWang
Contribution link: [apache/dolphinscheduler-website#456](https://github.com/apache/dolphinscheduler-website/pull/456)

**Comment by yangqiyu0306** [comment]
GitHub ID: yangqiyu0306
Contribution link:
#6907
#6857
#6843

**Comment by mgsky1** [comment]
GitHub ID: mgsky1
Contribution link:
[#6167](https://github.com/apache/dolphinscheduler/pull/6167)
[#6272](https://github.com/apache/dolphinscheduler/pull/6272)
[#442 webside](https://github.com/apache/dolphinscheduler-website/pull/442)

**Comment by shink** [comment]
GitHub ID: @shink
Contribution link: 
- [#5924](https://github.com/apache/dolphinscheduler/pull/5924)
- [#5817](https://github.com/apache/dolphinscheduler/pull/5817)
- [#438 website](https://github.com/apache/dolphinscheduler-website/pull/438)
- [#444 website](https://github.com/apache/dolphinscheduler-website/pull/444)

**Comment by didiaode18** [comment]
GitHub ID: didiaode18
Contribution link: #5200

**Comment by RedemptionC** [comment]
GitHub ID: Redemptionc

Contribution link: #3191

**Comment by zixi0825** [comment]
GitHub ID: zixi0825
Contribution link: #4283

**Comment by cxin-william** [comment]
GitHub ID: cxin-william

Contribution link: #6076

**Comment by wcmolin** [comment]
GitHub ID: wcmolin
Contribution link: [#7553](https://github.com/apache/dolphinscheduler/issues/7553#issue-1086780508)

**Comment by CtrlSorcerer** [comment]
GitHub ID: SunHao1234
Contribution link:[https://dolphinscheduler.apache.org/zh-cn/blog/Eavy_Info.html](url)

**Comment by eye-gu** [comment]
GitHub ID: ououtt
Contribution link: #6556

**Comment by zhuangchong** [comment]
GitHub ID: zhuangchong
Contribution link: #7318

**Comment by songjianet** [comment]
GitHub ID: songjianet
Contribution link: [#7781](https://github.com/apache/dolphinscheduler/pull/7781)

**Comment by wen-hemin** [comment]
GitHub ID: wen-hemin
Contribution link: #5498

**Comment by labbomb** [comment]
GitHub ID: labbomb
Contribution link:  #7674

**Comment by zh0122** [comment]
GitHub ID:  zh0122

Contribution link:
https://github.com/apache/dolphinscheduler/pull/4285
https://github.com/apache/dolphinscheduler-website/pull/213

**Comment by calvinjiang** [comment]
GitHub ID: calvinjiang
Contribution link: #7766

**Comment by Tandoy** [comment]
GitHub ID: Tandoy
Contribution link:
#6210
#6227

**Comment by boy-xiaozhang** [comment]
GitHub ID: boy-xiaozhang

Contribution link: [#7029 ](https://github.com/apache/dolphinscheduler/pull/7029)

**Comment by shangeyao** [comment]
GitHub ID: shangeyao

Contribution link: #7571  #6759

**Comment by jim-parsons** [comment]
GitHub ID: tangchunbo

Contribution link: 
#7782 #7506

**Comment by Eights-Li** [comment]
GitHub ID: Eights-Li
Contribution link:  https://github.com/apache/dolphinscheduler/commit/4d8d74ac1816e9ba11038d57cffb491074ce0a78

**Comment by GaoTianDuo** [comment]
GitHub ID: GaoTianDuo
Contribution link：https://github.com/apache/dolphinscheduler/issues/7061

**Comment by Yibo-Liu** [comment]
GitHub ID: Yibo-Liu
Contribution link：
https://github.com/apache/dolphinscheduler-website/pull/294
https://github.com/apache/dolphinscheduler/pull/4699

**Comment by yangzi1104** [comment]
Github ID: lyggyhmm
Contribution link: #6708

**Comment by c-f-cooper** [comment]
GitHub ID: c-f-cooper
Contribution link: [#7556](https://github.com/apache/dolphinscheduler/pull/4630)

**Reply by c-f-cooper** [reply]
https://github.com/apache/dolphinscheduler/pull/4627

**Reply by c-f-cooper** [reply]
https://github.com/apache/dolphinscheduler/pull/4525
https://github.com/apache/dolphinscheduler/pull/4312
https://github.com/apache/dolphinscheduler/pull/618

**Comment by yimaixinchen** [comment]
Github ID:yimaixinchen
Contribution link:#4795

**Comment by yhhmo** [comment]
Github ID:yhhmo
Contribution link:https://github.com/apache/dolphinscheduler/issues/57

**Comment by zhongjiajie** [comment]
GitHub ID: zhongjiajie
Contribution link: #7759, https://github.com/apache/dolphinscheduler-website/pull/611

**Reply by debrachena** [reply]
Hi there, I haven't received your post information here https://jinshuju.net/f/zDyzxp, please fill in the sheet before 1.19.2022.

**Comment by kamisangk** [comment]
GitHub ID: kamisangk
Contribution link:
[#4314](https://github.com/apache/dolphinscheduler/pull/4344)

**Comment by liurongshuo** [comment]
GitHub ID: liurongshuo
Contribution link:[#309](https://github.com/apache/dolphinscheduler-website/pull/309)

**Comment by songquanhe-gitstudy** [comment]
GitHub ID: songquanhe-gitstudy
Contribution link:
#2265
#2274

**Comment by LiuBodong** [comment]
GitHub ID: LiuBodong
Contribution link:
#6246
#6295
#6626

**Comment by Kyofin** [comment]
GitHub ID: huzk8
Contribution link:
#3968

**Comment by wuchunfu** [comment]
GitHub ID: [wuchunfu](https://github.com/wuchunfu)
Contribution link:
https://github.com/apache/dolphinscheduler/pull/3425
https://github.com/apache/dolphinscheduler/pull/3306
https://github.com/apache/dolphinscheduler/pull/3142
https://github.com/apache/dolphinscheduler/pull/1413
https://github.com/apache/dolphinscheduler/pull/917

**Comment by RichardStark** [comment]
GitHub ID: RichardStark
Contribution link: 
#6042
#6257
#6299
#6317

**Comment by ghost** [comment]
GitHub ID: enoughwell
Contribution link: #4638

**Comment by BenjaminWenqiYu** [comment]
GitHub ID: BenjaminWenqiYu
Contribution link: 
#6408

**Comment by brave-lee** [comment]
GitHub ID: JinyLeeChina
Contribution link: #7829

**Comment by KonsonChow** [comment]
GitHub ID: KonsonChow
Contribution link: [#5167](https://github.com/apache/dolphinscheduler/pull/5167)

**Comment by Yao-MR-zz** [comment]
GitHub ID:Yao-MR
Contribution link: 
     #5492
     #5906

**Comment by aaronlinv** [comment]
GitHub ID: aaronlinv
Contribution link:
#7687
[#516 website](https://github.com/apache/dolphinscheduler-website/pull/516)

**Comment by soreak** [comment]
GitHub ID: soreak
Contribution link:
[#5882](https://github.com/apache/dolphinscheduler/pull/5882)

**Comment by chenshuai1995** [comment]
GitHub ID：chenshuai1995
Contribution link：
[#4988](https://github.com/apache/dolphinscheduler/issues/4988)

**Comment by springmonster** [comment]
Github ID: springmonster
Contribution link:
https://github.com/apache/dolphinscheduler/pull/7798
https://github.com/apache/dolphinscheduler/pull/7676

**Comment by SbloodyS** [comment]
GitHub ID: SbloodyS
Contribution link:
#7551
#7465

**Comment by wangyizhi1** [comment]
Github ID: Wangyizhi1
Contribution link:  #7725

**Comment by echohlne** [comment]
Github ID: echohlne
Contribution link:
#5912
#5909

**Comment by andream7** [comment]
Github ID: andream7
Contribution link:[#6104](https://github.com/apache/dolphinscheduler/pull/6104)

**Comment by chengchuen** [comment]
Github ID: chengchuen
Contribution link:#7032

**Comment by cyx-code** [comment]
Github ID: cyx-code
Contribution link:[#5313](https://github.com/apache/dolphinscheduler/pull/5313)

**Comment by dongzl** [comment]
GitHub ID: dongzl
Contribution link: #7261

**Comment by felix-thinkingdata** [comment]
GitHub ID: felix-thinkingdata
Contribution link: https://github.com/apache/dolphinscheduler/pull/4724

**Comment by chengshiwen** [comment]
GitHub ID: chengshiwen
Contribution link: #5113

**Reply by debrachena** [reply]
Hi there, I haven't received your post information here https://jinshuju.net/f/zDyzxp, please fill in the sheet before 1.19.2022.

**Comment by ouyangyewei429** [comment]
GitHub ID: ouyangyewei
Contribution link: 
[#7420](https://github.com/apache/dolphinscheduler/pull/7420)
[#7411](https://github.com/apache/dolphinscheduler/pull/7411)
[#7404](https://github.com/apache/dolphinscheduler/pull/7404)
[#7308](https://github.com/apache/dolphinscheduler/pull/7308)
#7194 
[#7170](https://github.com/apache/dolphinscheduler/pull/7170)
[#7111](https://github.com/apache/dolphinscheduler/pull/7111)
#7050 
[#6669](https://github.com/apache/dolphinscheduler/pull/6669)

**Comment by Aaronoooooo** [comment]
GitHub ID: Aaronoooooo
Contribution link:  https://github.com/apache/dolphinscheduler/issues/57#issuecomment-1007996072

**Comment by qiuzhongbiao** [comment]
GitHub ID: qiu.zhongbiao
Contribution link:公众号排版

**Comment by JiangTChen** [comment]
GitHub ID: JiangTChen
Contribution link: #7877

**Comment by ShuoTiann** [comment]
Github ID: ShuoTiann
Contribution link: https://github.com/apache/dolphinscheduler/pull/7985

**Comment by z7658329** [comment]
Github ID: z7658329
Contribution link: 
#4481

**Comment by wangdazhong001** [comment]
Github ID: wangdahzong001
Contribution link: 
[#4843](https://github.com/apache/dolphinscheduler/issues/4843)

**Comment by Ylisen** [comment]
GitHub ID: Ylisen
Contribution link:
#4688
https://github.com/apache/dolphinscheduler-website/pull/281

**Comment by Hxssssss** [comment]
Github ID: Hxssssss
Contribution link:
#4449

**Comment by lfyee** [comment]
Github ID: ilfyee
Contribution link: [#1830](https://github.com/apache/dolphinscheduler/issues/1830)

**Comment by yaoyingzhe** [comment]
Github ID: yaoyingzhe
Contribution link: [#1830](https://github.com/apache/dolphinscheduler-website/pull/598)

**Comment by linyanbin666** [comment]
GitHub ID:  linyanbin666
Contribution link: #4474

**Comment by lianjie1229** [comment]
GitHub ID: lianjie1229
Contribution link: #1379

**Comment by xianweiY** [comment]
gitHub ID：yang100455
Contribution link: https://github.com/apache/dolphinscheduler/pull/6805

**Comment by lvshaokang** [comment]
GitHub ID: lvshaokang
Contribution link: #5785

**Comment by jixiang155** [comment]
GitHub ID: jixiang155
Contribution link: [#6423](https://github.com/apache/dolphinscheduler/issues/6423)

**Comment by jonwjh94** [comment]
GitHub ID: jonwjh94
Contribution link: #7671

**Comment by yh2388** [comment]
GitHub ID: yh2388
Contribution link: #2908

**Comment by XiaochenNan** [comment]
GitHub ID: XiaochenNan
Contribution link: [apache/dolphinscheduler-website#456](https://github.com/apache/dolphinscheduler/pull/8054)

**Comment by sparklezzz** [comment]
GitHub ID: sparklezzz
Contribution link: #7027

**Comment by 981433814** [comment]
GitHub ID: 981433814
Contribution link: #6655
